docs/UserGuide.adoc

```diff
@@ -217,12 +257,19 @@ Examples:
 Set the password of the app to `n3wP4sS`.
 // end::setpass[]
 
+[big red]#List of Parameters#:
+
+The old and new passwords of the application. +
```

Comment:
This doesn't follow the format established for the other commands.
Try using:

`CURRENT_PASSWORD`: The old password of the application.
`NEW_PASSWORD`: The new password.

Reply:
I don't want to do it this way because some people might think that these are prefixes.
Hence the very different presentation.

Reply:
OK, I have revised this.